addons/website/views/snippets/s_table_of_content.xml

```diff
@@ -15,7 +15,7 @@
                         <h1 data-anchor="true">Intuitive system</h1>
                         <h4>What you see is what you get</h4>
                         <p>
-                            Insert text styles like headers, bold, italic, lists and fonts with
+                            Insert text styles like headers, bold, italic, lists, and fonts with
```

Comment:
A comma before "and" ?

Reply:
Yes, I know... I learned that when I was working with the US team.
But in this case, it was mostly detected by my Grammarly app.
[Full explanation](https://www.grammarly.com/blog/comma-before-and/)

Reply:
Actually if I read the explanation right, the comma is useless :p but whatever, ok for me :)